lib/pluggy_elixir/config.ex

```diff
@@ -0,0 +1,83 @@
+defmodule PluggyElixir.Config do
+  @moduledoc """
+  PluggyElixir client configurations
+
+  To use `PluggyElixir` you must add into your config file `config.exs` the
+  required config:
+
+      config :pluggy_elixir,
+        client_id: "your-app-client-id",
+        client_secret: "your-app-client-secret",
```

Comment:
```suggestion
     ```
      config :pluggy_elixir,
        client_id: "your-app-client-id",
        client_secret: "your-app-client-secret"
      ```
```

It's possible to add a code block here? Not sure how would be the documentation output though

Reply:
HexDocs knows its a code block because the  indentation.